client/header/activity-panel/panels/stock/card.js

```diff
@@ -117,6 +135,17 @@ class ProductStockCard extends Component {
 
 	render() {
 		const { product } = this.props;
+		const { edited, editing } = this.state;
+		const { notifyLowStockAmount } = wcSettings;
+		const lowStockAmount = Number.isFinite( product.low_stock_amount )
+			? product.low_stock_amount
+			: notifyLowStockAmount;
+		const isLowStock = product.stock_quantity < lowStockAmount;
```

Comment:
Should this be `<=` to match https://github.com/woocommerce/woocommerce/blob/a247e08ad0b0fa93b8ece6ecd48b7b4d7ec432db/includes/admin/class-wc-admin-dashboard.php#L221 ?

Reply:
Good catch! Fixed in 682fe11c269403652397d51779c9dcc993503422.